gulp-core-build-sass/src/SassTask.ts

```diff
@@ -41,8 +47,9 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
     sassMatch: [
       'src/**/*.scss'
     ],
-    useCSSModules: false,
-    dropCssFiles: false
+    treatAllFilesAsCSSModules: false,
```

Comment:
Perhaps... I was on the fence about this. I had apparently misunderstood the meaning of this flag. I thought it had to be on to support css modules, but really it treats EVERYTHING as a css module, so I think it should be renamed.

We aren't using this flag anywhere. Are you?